src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -580,7 +584,99 @@ public void should_fail_when_classification_is_supplementary_evidence_with_ocr_d
                     RESPONSE_FIELD_ERRORS,
                     hasItem("The 'attach to case' event is not supported for supplementary evidence with OCR "
                         + "but not containing OCR data")
-        );
+                );
+    }
+
+    @Test
+    public void should_fail_when_classification_is_exception_and_exception_record_contains_pending_payments() {
+        CallbackRequest callbackRequest =
+            callbackRequestWith(
+                EVENT_ID_ATTACH_TO_CASE,
+                CLASSIFICATION_EXCEPTION,
+                "Yes", // awaiting payments DCN processing
+                false
+            );
+
+        given()
+            .body(callbackRequest)
+            .headers(userHeaders())
+            .post(CALLBACK_ATTACH_CASE_PATH)
+            .then()
+            .statusCode(200)
+            .body(
+                RESPONSE_FIELD_ERRORS,
+                hasItem("Cannot attach this exception record to a case because it has pending payments")
+            );
+    }
+
+    @Test
+    public void should_not_return_payments_error_when_supplementary_evidence_with_ocr_have_pending_payments() {
```

Comment:
it's not clear that this applies only to 'bulkscan' jurisdiction/service and that the rules on which classifications are allowed and which are not.
Is there are way to set up this in test in the 'given' stage?

Or at least providing the config on the test class? (extracting those tests to a separate class would come in handy here)

Reply:
I'll move these tests to a separate class and rename the tests

Reply:
As discussed yesterday, I'll refactor the tests in a separate PR. For now, I renamed the tests and added a comment for classification.